app/controllers/spree/admin/general_settings_controller_decorator.rb

```diff
@@ -8,6 +8,7 @@ module GeneralSettingsEditPreferences
       def edit
         super
         @preferences_general << :bugherd_api_key
+        @preferences_general << :user_guide_link
```

Comment:
I'm not sure about this, should I create a method below to fill `@preferences_general` and avoid these two lines ? Or something else ?